content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -6,5 +6,278 @@ weight: 1
 owner: istio/wg-networking-maintainers
 test: n/a
 ---
+{{< warning >}}
+Ambient is currently in alpha status, not intended for production use.
+{{< /warning >}}
 
-This page is under construction.
+This guide lets you quickly evaluate Istio ambient service mesh. These steps require you to have
+a {{< gloss >}}cluster{{< /gloss >}} running a
+[supported version](/docs/releases/supported-releases#support-status-of-istio-releases) of Kubernetes ({{< supported_kubernetes_versions >}}). You can use any supported platform, for
+example [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) or
+others specified by the
+[platform-specific setup instructions](/docs/setup/platform-setup/).
+
+Follow these steps to get started with ambient:
+
+1. [Download and install](#download)
+1. [Deploy the sample application](#bookinfo)
+1. [Adding your application to ambient](#addtoambient)
+1. [Secure application access](#secure)
+1. [Control traffic](#control)
+1. [Uninstall](#uninstall)
+
+## Download and install {#download}
+
+1.  Download the [alpha version of Istio](https://github.com/istio/istio/wiki/Dev-Builds) with support for ambient mesh.
+    If you don’t have a Kubernetes cluster, you can set up using
+    locally (e.g. using kind as below) or deploy one in Google or AWS Cloud:
+
+    {{< text bash >}}
+    $ kind create cluster --config=- <<EOF
+    kind: Cluster
+    apiVersion: kind.x-k8s.io/v1alpha4
+    name: ambient
+    nodes:
+    - role: control-plane
+    - role: worker
+    - role: worker
+    EOF
+    {{< /text >}}
+
+1.  The `ambient` profile is designed to help you get started with ambient mesh.
+    Install Istio with the `ambient` profile on your Kubernetes cluster, using
+    the `istioctl` downloaded above:
+
+    {{< text bash >}}
+    $ istioctl install --set profile=ambient
+    {{< /text >}}
+
+1.  After running the above command, you’ll get the following output that indicates
+    these five components are installed successfully!
+
+    {{< text plain >}}
+    ✔ Istio core installed
+    ✔ Istiod installed
+    ✔ CNI installed
+    ✔ Ingress gateways installed
+    ✔ Ztunnel installed
+    ✔ Installation complete
+    {{< /text >}}
+
+## Deploy the sample application {#bookinfo}
+
+You’ll use the sample [bookinfo application](/docs/examples/bookinfo/), which is part of
+your Istio download from previous steps. In ambient mode, you deploy applications to
+your Kubernetes cluster exactly the same way you would
+without Istio. This means you can have your applications running in your Kubernetes before
+you enable ambient mesh, and have them join the mesh without needing to restart or
+reconfigure your applications.
+
+{{< text bash >}}
```

Comment:
```suggestion
{{< warning >}}
Make sure the default namespace does not include the label `istio-injection=enabled` because when using ambient you do not want Istio to inject sidecars into the application pods.
{{< /warning >}}

{{< text bash >}}
```